lib/groonga/command/format/elasticsearch.rb

```diff
@@ -25,20 +25,30 @@ def initialize(name, arguments)
           @arguments = arguments
         end
 
-        def command_line
+        def command_line(options={})
           return if @name != "load"
 
           header = Hash.new
           body = Hash.new
           components = ""
+          elasticsearch_version = options[:version]
 
           sorted_arguments = @arguments.sort_by do |name, _|
             name.to_s
           end
           sorted_arguments.each do |name, value|
             case name
             when :table
-              header = {"index"=>{"_index"=>"groonga", "_type"=>"#{value}"}}
+              case elasticsearch_version
+              when 5, 6
```

Comment:
ES version 5, 6 and else clause are same.
It seems that this clause is needless.